mu/modes/python3.py

```diff
@@ -68,16 +80,13 @@ def start_kernel(self):
             os.environ[k] = v
         # Ensure the expected paths are in PYTHONPATH of the subprocess so the
         # kernel and Mu-installed third party applications can be found.
-        if "PYTHONPATH" not in os.environ:
-            paths = sys.path + [MODULE_DIR]
-            os.environ["PYTHONPATH"] = os.pathsep.join(paths)
-        if MODULE_DIR not in os.environ["PYTHONPATH"]:
-            # This is needed on Windows to ensure user installed third party
-            # packages are available in the REPL.
-            new_path = os.pathsep.join([os.environ["PYTHONPATH"], MODULE_DIR])
-            os.environ["PYTHONPATH"] = new_path
-        logger.info("REPL PYTHONPATH: {}".format(os.environ["PYTHONPATH"]))
+        if "PYTHONPATH" not in os.environ and self.pythonpath:
+            os.environ["PYTHONPATH"] = self.pythonpath
```

Comment:
So, if pythonpath is added in the constructor, is only used if the environmental variable is not set?

Reply:
Honestly, all the PYTHONPATH stuff is a relic of the previous implementation, but I didn't have the guts to remove it altogether.

Now that I've got a more stable test base I'll see if I can lose it altogether.

Reply:
Might be worth opening an issue to look into it after merging this PR, if this was the original implementation let's not delay this PR for it.